### ISSUE TITLE
error: Your local changes to the following files would be overwritten by merge:

### ISSUE THREAD

**Issue by kfinke** [typo3-themes/Vagrant.Themes#8]
Hi,
I want to pull the themes-extension, but I get this error:

![2015-01-07 09_23_00-mingw32__c_typo3-themes-box_project_typo3conf_ext_themes](https://cloud.githubusercontent.com/assets/281081/5642739/08693da8-964f-11e4-963c-d1cbfc779abd.png)

When doing a 'vagrant provision' I get some errors saying the same about some more files.

Example: 
==> default: --------------------------------------------------
==> default:  
==> default: TYPO3 Extension from themes
==> default:   From http://github.com/typo3-themes/themes.git
==> default:       Already there
==> default: error: Your local changes to the following files would be overwritten by merge:
==> default:    README.md
==> default:    project/typo3conf/LocalConfiguration.php
==> default:    project/typo3conf/PackageStates.php
==> default:    serverdata/data/sql/t3-latest.sql
==> default:    serverdata/etc/php5/apache2/php.ini
==> default:    serverdata/provision/bootstrap.sh
==> default: Please, commit your changes or stash them before you can merge.
==> default: Aborting
==> default: Updating 58d6f37..1c140e3
==> default: 58d6f37 Fixed Vagrantfile
==> default: e4e19fb Merge branch 'master' of https://github.com/typo3-themes/Vagrant.Themes
==> default: 57779c0 Fixed Vagrantfile

Then I did a 'git pull' again, but got the same error as before.

Thanks for any hint!
Kerstin

**Comment by pgampe** [comment]
Go into those folders, do a ``git status`` and post the output for us. Then do a ``git stash`` to revert to the lasted ``HEAD`` while saving the changes into to git stash. If you do want want those changes, please just do  a ``git reset --hard HEAD``.

If ``git status`` shows some files with a question mark, then please simply remove those files. Afterwards, please try again.

**Comment by kfinke** [comment]
Hi Philip, 

thanks for your reply.

Here is the output of git status:

--------------------

On branch master
Your branch is behind 'origin/master' by 14 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   ../../../../Vagrantfile
	modified:   ../../../../serverdata/provision/bootstrap.sh
	modified:   ../../../../serverdata/provision/install-extensions.sh
	modified:   ../../../../serverdata/provision/prepare.sh

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	../../../.idea/
	../../../fileadmin/blog/
	../../ENABLE_INSTALL_TOOL
	../../../vagrant/
	../../../../serverdata/provision/bootstrap - Kopie.sh
	../../../../vagrant/

no changes added to commit (use "git add" and/or "git commit -a")
-----------------
There were no files with question mark ...

Now I did a <code>git stash</code>, then <code>git pull</code> and it pulled the changes.

Now I got another error in the extension manager after updating the extension list:
"Version number in composer manifest of package "belayout_tsprovider" is missing or invalid"
So I had to delete the composer.json to get back in the extension manager.
And: I don't get the update button for "themes"-extension in the "manage-extension list".

One other thing seems strange to me: I have a folder "vagrant" in the themes-Extension, so: /project/typo3conf/ext/themes/vagrant. Is this alright or have I shuffled something in my folder structure?

So my state is: I can work now on my theme, but I'm little bit confused. Themes-extension is still version number 1.0.1 Should I update the themes-extension? If so, what's the best way?

Greetings
Kerstin
